src/features/smartSearch/components/SmartSearchDialog/FilterGallery/filterGalleryPattern.ts

```diff
@@ -123,10 +123,10 @@ export default function filterGalleryPattern(
     pattern = PATTERN_TEMPLATES.pattern15;
   }
 
-  pattern.background = pattern.background.replaceAll(
-    '$strongColor',
-    colors.strong
-  );
-  pattern.background = pattern.background.replaceAll('$paleColor', colors.pale);
-  return pattern;
+  return {
+    ...pattern,
+    background: pattern.background
+      .replaceAll('$strongColor', colors.strong)
+      .replaceAll('$paleColor', colors.pale),
+  };
```

Comment:
I think all this pattern making code needs to be replaced for something more useful/readable/sustainable - not antything to do with this pr, just thinking abou it now.